src/main/java/nl/knaw/huc/resussun/resources/ApiResource.java

```diff
@@ -88,6 +127,26 @@ private ServiceManifest createServiceManifest(ApiData api) {
         String.format("Dataset \"%s\" of \"%s\" OpenRefine Recon API", api.getDataSourceId(), api.getTimbuctooUrl()),
         "http://example.org/identifierspace", "http://example.org/schemaspace")
         .viewUrl(viewUrl)
-        .preview(new Preview(previewUrl, 400, 200));
+        .preview(new Preview(previewUrl, 400, 200))
+        .extend(
+            new Extend().proposeProperties(
+                urlHelperFactory.urlHelper(api.getDataSourceId()).template(),
+                "/extend/properties"
+            ).propertySetting(new ExtendPropertySetting<Integer>(
+                "limit",
+                "Limit",
+                "number",
+                0,
+                "Maximum number of values to return per row (0 for no limit)"
+            )).propertySetting(new ExtendPropertySetting<String>(
+                    "content",
+                    "Content",
+                    "select",
+                    "literal",
+                    "Content type: ID or literal"
+                ).choice("id", "ID")
+                 .choice("literal", "Literal")
+            )
```

Comment:
These property settings are are declared in the model, but ignored if included in the actual request.